astropy/modeling/math_functions.py

```diff
@@ -0,0 +1,67 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import warnings
+import numpy as np
+
+from astropy.modeling.core import Model
+
+
+trig_ufuncs = ["sin", "cos", "tan", "arcsin", "arccos", "arctan", "arctan2",
+               "hypot", "sinh", "cosh", "tanh", "arcsinh", "arccosh",
+               "arctanh", "deg2rad", "rad2deg"]
+
+
+math_ops = ["add", "subtract", "multiply", "divide", "logaddexp", "logaddexp2",
```

Comment:
Should we expose add, subtract, multiply, divide, and power since they are already implicitly in modeling?

Reply:
These take two inputs and add, ..., them, e.g. two arrays.
Model.add adds the result of two Model.evaluate functions.
The former can be achieved with the latter but is way more handy to have it as a ufunc.